torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2012,6 +2012,15 @@ def sample_inputs_isclose(
         yield SampleInput(lhs, args=(rhs,),
                           kwargs=dict(op_kwargs, rtol=rtol, atol=atol, equal_nan=equal_nan))
 
+def sample_inputs_linalg_vecdot(op_info, device, dtype, requires_grad, **kwargs):
+    yield from sample_inputs_binary_pwise(op_info, device, dtype, requires_grad)
```

Comment:
Not in love with reusing the elementwise binary sample inputs for this op -- wouldn't it be simple to generate some examples with and without batch dims, an empty tensor, and dim of the default, -2, 1, and 0?